content/english/articles/guides/remote-chain/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Connecting to a remote Gno chain (from scratch)
+publishDate: 2025-01-20T08:00:00-01:00
+translationKey: "remote-chain"
```

Comment:
Just for info, it's also possible to add a `metaTitle` and a `metaDescription` here if we want to decouple them from the `title` and the `summary`.